README.md

```diff
@@ -467,14 +467,32 @@ With the use of the [Home Assistant RESTful][ha-rest] integration, one should
 be able to grab some interesting data from this add-on.
 
 Example:
-
+Set the port in the add-on settings; for example: 6789 and your IP of your Home Assistant instance; the hassio.local didn't work for me.
+Add a rest sensor:
 ```yaml
-sensor:
-  - platform: rest
-    resource: http://a0d7b954-wireguard
+platform: rest
+name: wireguard_sensor
+resource: http://192.168.3.72:6789/
+value_template: 'TEST'
+json_attributes:
+  - ipad
+  - mac
+  - telefoon
 ```
-
-At this moment, we do not have template or examples on how this could be
+The value_template: 'TEST' is for having a state; if you don't do this you will reach the 255 characters state limit.
+For each peers add a json_attributes entry.
+For each peer you will get a attribute such as: 
+```json { "endpoint": "172.30.32.1:47004", "latest_handshake": 0, "transfer_rx": 48699004, "transfer_tx": 625289972 }```
+
+Having bytes isn't so useful; that's why we can change it with filesizeformat and a template:
+platform: template
+sensors:
+  ipad_sent:
+  value_template: '{{ states.sensor.wireguard_sensor.attributes["ipad"]["transfer_tx"] |filesizeformat}}'
+Check the end result:
+![Example picture](https://i.imgur.com/tRINGjC.png)
+
+At this moment, we have only a few templates or examples on how this could be
```

Comment:
```suggestion
For each peer you will get a attribute such as: 
```json { "endpoint": "172.30.32.1:47004", "latest_handshake": 0, "transfer_rx": 48699004, "transfer_tx": 625289972 }```

It is possible to generate template sensors for all peers with the following code pasted in the template editor (change peers to match your configuration):
```jinja
{% set peers = ["iphone", "ipad", "mac"] %}
{% set entities = ["endpoint", "latest_handshake", "transfer_rx", "transfer_tx"] %}
{{my_json}}
  - platform: template
    sensors:
{%- for peer in peers %}{% for entity in entities %}
      wireguard_{{ peer|lower }}_{{ entity|lower }}:
        friendly_name: {{ peer|title }} {{ entity | title }}
        value_template: {{"\"{{ state_attr('sensor.wireguard_peers', '"}}{{- peer }}')['{{ entity }}']}}"
{%- endfor %}{% endfor %}
```
Which will generate the following template sensors:
```yaml
  - platform: template
    sensors:
      wireguard_iphone_endpoint:
        friendly_name: Iphone Endpoint
        value_template: "{{ state_attr('sensor.wireguard_peers', 'iphone')['endpoint']}}"
      wireguard_iphone_latest_handshake:
        friendly_name: Iphone Latest_handshake
        value_template: "{{ state_attr('sensor.wireguard_peers', 'iphone')['latest_handshake']}}"
      wireguard_iphone_transfer_rx:
        friendly_name: Iphone Transfer_rx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'iphone')['transfer_rx']}}"
      wireguard_iphone_transfer_tx:
        friendly_name: Iphone Transfer_tx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'iphone')['transfer_tx']}}"
      wireguard_ipad_endpoint:
        friendly_name: Ipad Endpoint
        value_template: "{{ state_attr('sensor.wireguard_peers', 'ipad')['endpoint']}}"
      wireguard_ipad_latest_handshake:
        friendly_name: Ipad Latest_handshake
        value_template: "{{ state_attr('sensor.wireguard_peers', 'ipad')['latest_handshake']}}"
      wireguard_ipad_transfer_rx:
        friendly_name: Ipad Transfer_rx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'ipad')['transfer_rx']}}"
      wireguard_ipad_transfer_tx:
        friendly_name: Ipad Transfer_tx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'ipad')['transfer_tx']}}"
      wireguard_mac_endpoint:
        friendly_name: Mac Endpoint
        value_template: "{{ state_attr('sensor.wireguard_peers', 'mac')['endpoint']}}"
      wireguard_mac_latest_handshake:
        friendly_name: Mac Latest_handshake
        value_template: "{{ state_attr('sensor.wireguard_peers', 'mac')['latest_handshake']}}"
      wireguard_mac_transfer_rx:
        friendly_name: Mac Transfer_rx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'mac')['transfer_rx']}}"
      wireguard_mac_transfer_tx:
        friendly_name: Mac Transfer_tx
        value_template: "{{ state_attr('sensor.wireguard_peers', 'mac')['transfer_tx']}}"
```

```